awesome.js

```diff
@@ -1,23 +1,40 @@
-// Book Class: Represents a Book
-class Book {
-  constructor(title, author, ) {
-    this.title = title;
-    this.author = author;
-  }
+// Book object: Represents a Book
+function Book(title, author) {
+  this.title = title;
+  this.author = author;
 }
 
-// UI Class: Handle UI Tasks
-class UI {
-  static displayBooks() {
+const Store = {
+  getBooks() {
+    let books;
+    if (localStorage.getItem('books') === null) {
+      books = [];
+    } else {
+      books = JSON.parse(localStorage.getItem('books'));
+    }
+
+    return books;
+  },
+
+  addBook(book) {
+    const books = Store.getBooks();
+    books.push(book);
+    localStorage.setItem('books', JSON.stringify(books));
+  },
+
+};
+
+const UI = {
```

Comment:
Good work so far , but I would suggest that you create only one class Books to manage your books, and call local storage only once when you initialize your books instance inside books class constructor rather than getting books from localstorage each time we call display books, and when adding a new book we update localstorage as well from addNewBook method of our class , and same when removing